src/v/datalake/translation/partition_translator.cc

```diff
@@ -303,11 +303,6 @@ partition_translator::do_translation_for_range(
     const auto& ntp = _partition->ntp();
     auto remote_path_prefix = remote_path{
       fmt::format("{}/{}/{}", iceberg_data_path_prefix, ntp.path(), _term)};
-    lazy_abort_source las{[this] {
-        return can_continue() ? std::nullopt
```

Comment:
Just curious, this used to account for term changes. Is that still covered with `_as`?

Reply:
I think the answer is that it'll be up to the scheduler/executor to abort during term changes via this abort source

Reply:
yeah i believe that's right

Reply:
yes true. An abort source is passed to this method that is triggered on term change too.